lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reuse-existing-identity.yaml

```diff
@@ -23,8 +23,33 @@ states:
       next:
         targetState: UPDATE_DETAILS_PAGE
 
+  UPDATE_IDENTITY_START:
+    events:
+      next:
+        targetState: UPDATE_IDENTITY_BEFORE_IDENTITY_REUSE_PAGE
+
   # Journey states
 
+  UPDATE_IDENTITY_BEFORE_IDENTITY_REUSE_PAGE:
+    response:
+      type: process
+      lambda: store-identity
+      lambdaInput:
+        identityType: UPDATE
```

Comment:
This is a bit TBD, but at the moment this will always be a `NEW` identity - there aren't any update journeys that support F2F.

(If we start supporting it in future, we'll need to consider this more carefully..!)